crates/builder/src/tasks/submit.rs

```diff
@@ -198,7 +195,8 @@ where
 
         // If configured with a local signer, we use it. Otherwise, we ask
         // quincey (politely)
-        let signed = if let Some(signer) = &self.config.local_sequencer_signer {
+        let signed = if let Some(sequencer_key) = &self.config.sequencer_key {
```

Comment:
This reconnects the signer on every new inbound, which is probably not what we want. We should instead instantiate the signer in `spawn` and pass an `Option<LocalOrAws>` into `handle_inbound` as an argument

Reply:
and like above, i'd like to see 

impl BuilderConfig {
    /// Connect the configured sequencer signer 
    pub async fn connect_sequencer_signer(&self) -> Option<Result<LocalOrAws, ConfigError>>;
}

Reply:
https://github.com/init4tech/zenith-rs/pull/12/commits/8b0e0d49f02fe66f0cd3fb9dab3876731fc519cb